### DISCUSSION TITLE
Автообновление?

### DISCUSSION THREAD

**Discussion by Mintedan** [cawa-93/anime-library#137] [Вопросы и ответы] [answered]
В какой версии появилось автообновление и можно ли будет его отключить? я так то не против но знатно офигел когда при закрытии проги она захотела что то изменить у меня на пк

**Comment by cawa-93** [comment, marked as answer]
Автообновление было с самых первых версий.

Отключить его нельзя.

**Reply by Mintedan** [reply]
хех а я все это время в ручную устанавливал, как только видел тут новую версию. но ты подумай о функции отключения ну или хотя бы уведомлении о выходе новой версии.

**Reply by cawa-93** [reply]
Уведомление приходит, когда новая версия доступна и готова к установке

**Reply by Mintedan** [reply]
странно я не заметил посмотрю при следующем обновлении
